scaleway/resource_rdb_acl.go

```diff
@@ -216,14 +216,22 @@ func rdbACLExpand(data []interface{}) ([]*rdb.ACLRuleRequest, error) {
 	var res []*rdb.ACLRuleRequest
 	for _, rule := range data {
 		r := rule.(map[string]interface{})
-		ip, err := expandIPNet(r["ip"].(string))
-		if err != nil {
-			return res, err
+
+		aclRule := &rdb.ACLRuleRequest{}
```

Comment:
Can we end up adding empty aclRule ? Is that normal? Or an acl with only a description and no ip

Reply:
Thank you for the feedback. I just updated the helper